CLOUD-209-sync-options-updated-operator-topic (#158)
percona/percona-xtradb-cluster-operator: configure/operator.md

| Key                            | Value Type | Example   | Description |
|--------------------------------|------------|-----------|-------------|
|size                            | int        | `3`       |  The size of the Percona XtraDB Cluster, must be >= 3 for [High-Availability](hhttps://www.percona.com/doc/percona-xtradb-cluster/5.7/intro.html) |
| allowUnsafeConfigurations      | string     | `false`   | Prevents users from configuring a cluster with unsafe parameters |

> |image                           | string     |`perconalab/percona-xtradb-cluster-operator:0.4.0-pxc` | Percona XtraDB Cluster docker image to use                                                                     |

Reviewer: ```suggestion
|image                           | string     |`percona/percona-xtradb-cluster-operator:1.0.0-pxc` | Percona XtraDB Cluster docker image to use                                                                     |
```

|resources.limits.memory         | string     | `1G`      | [Kubernetes Memory limit](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#resource-requests-and-limits-of-pod-and-container) for a PXC container |
|resources.limits.cpu            | string     | `1`       | [Kubernetes CPU limit](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#resource-requests-and-limits-of-pod-and-container) for a PXC container |
|nodeSelector                    | label      | `disktype: ssd`        | The [Kubernetes nodeSelector](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector) constraint|
|affinity.topologyKey            | string     |`kubernetes.io/hostname`| The [Operator topologyKey](./constraints) node anti-affinity constraint|
|affinity.advanced               | subdoc     |           | If available, it makes [topologyKey](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#inter-pod-affinity-and-anti-affinity-beta-feature) node affinity constraint to be ignored |

> |affinity.tolerations                     | subdoc     | `node.alpha.kubernetes.io/unreachable` | The [Kubernetes Pod tolerations] (https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/#concepts)            |

Reviewer 2: ```suggestion
|tolerations                     | subdoc     | `node.alpha.kubernetes.io/unreachable` | The [Kubernetes Pod tolerations] (https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/#concepts)            |
```

|--------------------------------|------------|-----------|-------------|
|enabled                         | boolean    | `true`    | Enables or disables [load balancing with ProxySQL](https://www.percona.com/doc/percona-xtradb-cluster/5.7/howtos/proxysql.html) [Service](https://kubernetes.io/docs/concepts/services-networking/service/) |
|size                            | int        | `1`       | The number of the ProxySQL daemons [to provide load balancing](https://www.percona.com/doc/percona-xtradb-cluster/5.7/howtos/proxysql.html), must be = 1 in current release|

> |image                           | string     |`pperconalab/percona-xtradb-cluster-operator:0.4.0-proxysql` | ProxySQL docker image to use |

Reviewer: ```suggestion
|image                           | string     |`percona/percona-xtradb-cluster-operator:1.0.0-proxysql` | ProxySQL docker image to use |
```

|volumeSpec.persistentVolumeClaim.storageClassName | string     | `standard`| The [Kubernetes Storage Class](https://kubernetes.io/docs/concepts/storage/storage-classes/) to use with the ProxySQL [Persistent Volume Claim](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims)            |
|volumeSpec.persistentVolumeClaim.accessModes | array      | `[ "ReadWriteOnce" ]` | [Kubernetes Persistent Volume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) access modes for ProxySQL  |
|volumeSpec.resources.requests.storage | string     | `2Gi`     | The [Kubernetes Persistent Volume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) size for ProxySQL                             |
| podDisruptionBudget.maxUnavailable   | int  | `1`    | [Kubernetes Disruption Budget](https://kubernetes.io/docs/tasks/run-application/configure-pdb/) The number of pods unavailable after eviction| 
| podDisruptionBudet.minAvailable      | int  | `0`   | [Kubernetes Disruption Budget](https://kubernetes.io/docs/tasks/run-application/configure-pdb/) The number of pods available after eviction |

> 

Reviewer: why so many new lines?

Author: Deleted empty lines.

| Key                            | Value Type | Example   | Description |
|--------------------------------|------------|-----------|-------------|

> |image                           | string     | `perconalab/percona-xtradb-cluster-operator:0.4.0-backup` | Percona XtraDB Cluster docker image to use for the backup functionality                                                                       |

Reviewer: ```suggestion
|image                           | string     | `percona/percona-xtradb-cluster-operator:1.0.0-backup` | Percona XtraDB Cluster docker image to use for the backup functionality                                                                       |
```

Author: Updated in current version.

|storages.type                   | string     | `s3`      | Type of the cloud storage to be used for backups. Currently only `s3` and `filesystem` types are supported |
|storages.s3.credentialsSecret   | string     | `my-cluster-name-backup-s3`| [Kubernetes secret](https://kubernetes.io/docs/concepts/configuration/secret/) for backups. It should contain `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` keys. |
|storages.s3.bucket              | string     |           | The [Amazon S3 bucket](https://docs.aws.amazon.com/en_us/AmazonS3/latest/dev/UsingBucket.html) name for backups     |
|storages.s3.region              | string     |`us-east-1`| The [AWS region](https://docs.aws.amazon.com/en_us/general/latest/gr/rande.html) to use. Please note **this option is mandatory** not only for Amazon S3, but for all S3-compatible storages.|
|storage.s3.endpointUrl | string|           | The endpoint URL of the S3-compatible storage to be used (not needed for the original Amazon S3 cloud) |

> |storage.type                    | string    | `filesystem` | persistent volume type |

Reviewer: duplicate, see line 88, 103

Author: One is for the persistentvolume - added text to make the distinction.

|storages.persistentVolumeClaim.storageClassName | string | `standard`| Set the [Kubernetes Storage Class](https://kubernetes.io/docs/concepts/storage/storage-classes/) to use with the PXC backups [Persistent Volume Claim](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims) for the `filesystem` storage type                    |
|storages.persistentVolumeClaim.accessModes | array | ["ReadWriteOnce"] | The [Kubernetes Persistent Volume access modes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes) |

> |storage | string | `6Gi`| the storage size |

Reviewer: what option is it?

Author: persistentvolume storage - added text to make the distinction.
